src/validators/satelliteIdValidator.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from 'express'
+import { assert, pattern, string } from 'superstruct'
+
+export const validateSatelliteIdRoute = (req: Request, res: Response, next: NextFunction):
+    //ReturnType<NextFunction> | ReturnType<typeof response.sendStatus> => {
```

Comment:
Remove this commented-out line. You can always look through Git history to see old code. By removing unused code, you will keep your codebase clean.